dev-util/gn/gn-01463.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="A meta-build system that generates build files for Ninja"
+DESCRIPTION="GN is a meta-build system that generates build files for Ninja"
+HOMEPAGE="https://gn.googlesource.com/"
+COPYRIGHT="2008-2018 The Chromium Authors."
+LICENSE="BSD (3-clause)"
+REVISION="1"
+srcGitRev="091169beda92aff21ded3e9dfb392ea12b10c0e0"
+SOURCE_URI="https://github.com/return/gn/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="e17c58a90e26212c74cd527b1a10d315ede15a64c9cf0755b4ef78418c379b76"
+SOURCE_DIR="gn-$srcGitRev"
+PATCHES="gn-01463.patchset"
```

Comment:
Suggestion:
```diff
 SOURCE_URI="https://github.com/return/gn/archive/$srcGitRev.tar.gz"
 CHECKSUM_SHA256="e17c58a90e26212c74cd527b1a10d315ede15a64c9cf0755b4ef78418c379b76"
+SOURCE_FILENAME="gn-$portVersion-$srcGitRev.tar.gz"
 SOURCE_DIR="gn-$srcGitRev"
-PATCHES="gn-01463.patchset"
+PATCHES="gn-$portVersion.patchset"
```

> Are there any problems preventing this recipe being merged?

Probably not :)
It's just that we have a lot of PRs (lucky us!) and we forget to review them all...